nservicebus/pipeline/features.md

```diff
@@ -16,21 +16,21 @@ related:
  - samples/startup-shutdown-sequence
 ---
 
-While NServiceBus provides some simple interfaces to plug in the code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
+While NServiceBus provides interfaces to plug in code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
```

Comment:
> distribute

Not sure what distributing extensions means in this context? e.g. as a nuget package/library? should we spell it out?